src/components/Nav.js

```diff
@@ -0,0 +1,51 @@
+import { Link, NavLink, useLocation } from "react-router-dom";
+import "../styles/common.css";
+import styles from "../styles/Nav.module.css";
+import logoImg from "../image/pandaMarketLogo.png";
+
+function getLinkStyle({ isActive }, currentPath) {
+  let color = isActive ? "#3692FF" : "#4B5563";
+  if (currentPath === "/additem") {
+    color = "#3692FF";
+  }
+  return {
+    textDecoration: "none",
+    color: color,
+  };
+}
```

Comment:
```suggestion
const activeColor = "#3692FF";
const defaultColor = "#4B5563";

const getLinkStyle = ({ isActive }, currentPath) => {
  const color = (isActive || currentPath === "/additem") ? activeColor : defaultColor;
  
  return {
    textDecoration: "none",
    color,
  };
};
```

현재는 로직이 두 군데로 나뉘어져 있고, 그에 따라 `#3692FF`라는 컬러도 각각 두 군데에 작성되어 있습니다.
컬러를 상수화 하여 공통적인 값을 사용하고, 로직도 더 직관적으로 변경해주시면 유지보수와 가독성에 좋을 것 같습니다!